southpark-incremental-indexing/tests/distributed/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN git clone https://github.com/jina-ai/jinad.git && \
 
 RUN python -c "from transformers import DistilBertModel, DistilBertTokenizer; x='distilbert-base-cased'; DistilBertModel.from_pretrained(x); DistilBertTokenizer.from_pretrained(x)"
 
-COPY . /
```

Comment:
is this being moved? I do not like this so strange workdir reference. Where is this file?